### PR TITLE
Improvements from Adam's review.

### PR DESCRIPTION
**Pull request by pennello**
[OTPL-2351][1]

cf. https://github.com/opentable/otj-filterorder/pull/1

[1]: https://opentable.atlassian.net/browse/OTPL-2351

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,15 @@ The filter order dependency resolver accepts dependency declarations in
 the form of injected beans and reorders servlet filters in order to
 accord to the desired dependencies.
 
-See the classes' Javadocs for usage.
+Here is an sample use.  In it, B is said to depend on A, so in the final
```
Comment (fsosa) [inline README.md]: `a sample`

Comment (pennello) [reply README.md]: Oops, thanks.  Fixed in 109027bd7282ab11b577be81853556080351b8fc.

Comment (pennello) [reply README.md]: Further improved based on in-person chatting with @fsosa in 8cd00a835574d91338003878780f6c82bef35ea6.

**Comment by fsosa** [comment]
bitmoji

**Comment by pennello** [comment]
@fsosa is going to release this, including the Sonatype OSS procedure.

**Pull request merged**